README.md

```diff
@@ -40,7 +40,15 @@ Remember to run `git submodule update --init` even after you checked out a new b
 ### II.1. Activate Spack in your current shell
 
 ```
-source spack/share/spack/setup-env.sh
+source thisfairsoft.sh
+```
+
+Verify that the `spack` command works and lists the correct FairSoft package [repository](https://spack.readthedocs.io/en/latest/repositories.html).
+
+```
+$ spack repo list
+==> 1 package repository.
+fairsoft    ~/FairSoft
```

Comment:
```suggestion
fairsoft    ~/FairSoft
builtin     ~/FairSoft/spack/var/spack/repos/builtin
```

Reply:
(in my branch it was correct because it pointed to a spack with https://github.com/FairRootGroup/spack/commit/6905640453b253b23c2effbea3ac685730e91a79)